### PR TITLE
New Drop - Use Azure Monitor Agent on Windows Arc-enabled Server

### PR DESCRIPTION
**Pull request by mrhoads**
This drop is the Windows-equivalent of a Drop for using the Azure Monitor Agent on an existing Arc-enabled server.

### CODE REVIEW

File: drops/deploy_ama_to_windows_arc_server.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+    "Source": "https://github.com/Azure/arc_jumpstart_drops/tree/main/tutorial_guide/deploy_ama_to_windows_arc_server",
+    "Type": "tutorial_guide",
+    "Difficulty": "Beginner",
+    "ProgrammingLanguage": [
+      "Bash"
+    ],
+    "Products": [
+      "Arc",
+      "Arc-enabled Servers"
```
Comment (fcabrera23) [inline drops/deploy_ama_to_windows_arc_server.json:19]: Maybe add Windows?

File: drops/deploy_ama_to_windows_arc_server.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      {
+        "Name": "Mike Rhoads",
+        "Link": "https://www.linkedin.com/in/mike-rhoads/"
+      }
+    ],
+    "Source": "https://github.com/Azure/arc_jumpstart_drops/tree/main/tutorial_guide/deploy_ama_to_windows_arc_server",
+    "Type": "tutorial_guide",
+    "Difficulty": "Beginner",
+    "ProgrammingLanguage": [
+      "Bash"
```
Comment (fcabrera23) [inline drops/deploy_ama_to_windows_arc_server.json:15]: Is this bash or powrshell?

File: tutorial_guide/deploy_ama_to_windows_arc_server/_index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+Using an Azure Resource Manager (ARM) template, you'll deploy the following resources:
+- A Log Analytics Workspace
+- A data collection rule
+- A data collection rule association
+- The Azure Monitor Agent extension for your existing Arc-enabled server
+
+To deploy, run:
+
+```
```
Comment (fcabrera23) [inline tutorial_guide/deploy_ama_to_windows_arc_server/_index.md:41]: add language

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
Please review comments

File: drops/deploy_ama_to_windows_arc_server.json (hunk 2 above)
Comment (mrhoads) [reply drops/deploy_ama_to_windows_arc_server.json:15]: I'm just showing the shell command to deploy the data collection rule so am left this at bash

**Pull request merged**